lib/openshift/collector.rb

```diff
@@ -5,7 +5,7 @@
 
 module Openshift
   class Collector
-    def initialize(source, openshift_host, openshift_port, openshift_token, default_limit: 100, poll_time: 30)
+    def initialize(source, openshift_host, openshift_port, openshift_token, default_limit: 50, poll_time: 30)
```

Comment:
The limits can actually be overridden per entity_type if you want to just drop the limit for container images?  More efficient to send larger batches if we are able to.

Reply:
I think for now, it doesn't matter much. I'd rather fix this later by the automatic learning of the number of entities.